app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -250,6 +250,7 @@ class BrowserToolbarView(
         private const val TOOLBAR_ELEVATION = 16
     }
 
+    @Suppress("ComplexCondition")
     private fun ToolbarMenu.Item.performHapticIfNeeded(view: View) {
         if (this is ToolbarMenu.Item.Reload && this.bypassCache ||
```

Comment:
This warning goes away if this is turned into a when instead of an if, but imo its not really anymore readable